### PR TITLE
Turkey Major Fix

### PR DESCRIPTION
**Pull request by erdemug**
- I couldn't think an easy way to remove Arabic clothes and fez so European heritage is added instead of Middle Eastern (I know that wearing a fez or turban reminds everyone of Ottoman culture, but in 1950 it was forbidden to wear them in Turkey. There's even a chapter in The Little Prince about it :))
- New names are added.

### CODE REVIEW

**Comment by erdemug** [comment]
- Capital city is fixed (although there is much to do, I'm planning to finish later. I thought capital city issue is much more important so I changed that.)
- City names are fixed (some of the city names were old or Ottoman version)
- Translations are added

**Comment by settintotrieste** [comment]
@erdemug Thanks! I've managed to find a way to remove the fez by editing the triggers for clothing so there's no need to change the Turkish heritage trait now.

**Comment by erdemug** [comment]
> @erdemug Thanks! I've managed to find a way to remove the fez by editing the triggers for clothing so there's no need to change the Turkish heritage trait now.

I've fixed it! Thanks.

File: common/cultures/00_cultures.txt [hunk 1]
```diff
@@ -6473,150 +6473,82 @@ turkish={
...
+	}
+	regal_last_names = {
+		Osmanoglu
 	}
 
 	ethnicities = {
 		1 = arab
 	}
-	graphics = mideast_indian
+	graphics = european
```
Comment (settintotrieste) [inline common/cultures/00_cultures.txt:6551]: I think you might have missed this out in your changes?

**Comment by settintotrieste** [comment]
@erdemug Thanks!

**Pull request merged**